resources/backend.rb

```diff
@@ -0,0 +1,98 @@
+#
+# Cookbook Name:: sensu-go
+# Resource:: backend
+#
+# Copyright:: 2018 Sensu, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+# LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+# WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+resource_name :sensu_backend
+
+property :version, String, default: 'latest'
+property :repo, String, default: 'sensu/nightly'
+property :config_home, String, default: '/etc/sensu'
+property :config, Hash, default: { "state-dir": '/var/lib/sensu' }
+
+action :install do
+  packagecloud_repo new_resource.repo do
+    type value_for_platform_family(
+      %w(rhel fedora amazon) => 'rpm',
+      'default' => 'deb'
+    )
+  end
+
+  package 'sensu-backend' do
+    action :install
+    version new_resource.version unless new_resource.version == 'latest'
+  end
+
+  # TODO: Remove this sad monkey patch if upstream PR is accepted
```

Comment:
Looks like we remove the sad monkey patch as the upstream fix has been accepted: https://github.com/sensu/sensu-go/pull/1354

Reply:
I'll remove it and push the changes in a few minutes